cosmoz-treenode-navigator.js

```diff
@@ -19,8 +19,9 @@
 			},
 			dataPlane: {
 				type: Array,
-				computed: '_computeDataPlane(_searching, inputValue, _renderedLevel, _openNodeLevelPathParts, data)',
-				notify: true
+				value: function() { return []; },
```

Comment:
Yes, true. This is still from a version of mine with observing and then setting dataPlane but returned to computed one.